test/e2e/components/vl-input-field.js

```diff
@@ -1,6 +1,32 @@
 const { VlElement } = require('vl-ui-core').Test;
+const { VlFormValidation } = require('vl-ui-form-validation').Test;
 
-class VlInputField extends VlElement {  
+class VlInputField extends VlFormValidation { 
+    
+    async setInputValue(content) {
```

Comment:
Ik zou `setInputValue` vervangen door `setValue` omdat het 'evident' is dat het de value van het input veld is.